.github/actions/launchable/setup/action.yml

```
@@ -264,10 +264,11 @@ runs:
         shell: bash
         working-directory: ${{ inputs.srcdir }}
         post: |
-          [[ "${test_all_enabled}" = "true" ]] && \
+          if [[ "${test_all_enabled}" == "true" ]]; then \
             launchable record tests \
               --session "$(cat "${test_all_session_file}")" \
-              raw "${test_report_path}" || true
+              raw "${test_report_path}" \
+          fi
```

Do not escape the newline just before `fi`.
```suggestion
              raw "${test_report_path}"
          fi
```